syntax = "proto3";

package antibiogram_ocr;

message ImageRequest {
  bytes image_data = 1;
}

message OCRResponse {
    string text = 1;
    string content_type = 2;  // Campo content_type
}


service OCRService {
  rpc PerformOCR (ImageRequest) returns (OCRResponse);
}
